src/content/en/basic-workflows/z-image-turbo.md

```
@@ -12,15 +12,19 @@ hero:
 tags: []
 ---
 
-## What is Z-Image?
+## What is Z-Image-Turbo?
 
 Z-Image is a **family of image generation models** by Alibaba / Tongyi-MAI.
 
-- Z-Image-Base: Base model (Unreleased)
-- Z-Image-Turbo: Photo-realistic text2image distilled from Base for few-step generation
-- Z-Image-Edit: Model for editing (Unreleased)
+![](https://gyazo.com/569a829f2dbd9021bfdecf6d1e3267b9){gyazo=image}
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)